### PR TITLE
Feature/modules projects

### PR DESCRIPTION
**Pull request by m1073496**
### What does this PR do?
* Creates ModuleNav and ProjectNav components, adds them to Home.vue
* Styles these components

### Anything specific the reviewer should look at?
* File structure, component structure

Closes #

### CODE REVIEW

File: src/components/ModuleNav.vue [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  <section class="left-section">
+    <router-link to="/mod-1" class="module">Module 1</router-link>
+    <router-link to="/mod-2" class="module">Module 2</router-link>
+    <router-link to="/mod-3" class="module">Module 3</router-link>
+    <router-link to="/mod-4" class="module">Module 4</router-link>
+  </section>
+</template>
+
+<script>
+</script>
```
Comment (knees4bees) [inline src/components/ModuleNav.vue:11]: I noticed these `<script>` tags don't have anything inside them. Does anyone know if it's best practice to have these here even if there's not content for them? Or should they be deleted?

File: src/components/ModuleNav.vue [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  display: grid;
+  grid-template-rows: repeat(4, 1fr);
+  margin: 0;
+  text-align: center;
+  height: 70px;
+  line-height: 70px;
+
+  &:hover {
+    text-decoration: underline;
+    background-color: $turing-blue;
```
Comment (knees4bees) [inline src/components/ModuleNav.vue:26]: Nice!

File: src/components/ProjectNav.vue [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: 1fr;
+}
+
+a.project {
+  border: 1px solid black;
+  padding: 10px;
+  text-align: center;
+
+  &:hover {
```
Comment (knees4bees) [inline src/components/ProjectNav.vue:29]: Nice use of nesting here. I need to remember to do that.

**Review by knees4bees — APPROVED** [review]
This looks great, Jessica! Nice work.

File: src/components/ModuleNav.vue (hunk 1 above)
Comment (m1073496) [reply src/components/ModuleNav.vue:11]: Nice catch! I think we will delete them going forward!

**Pull request merged**